src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1313,7 +1396,7 @@ static bool addTriggeredSubscriptions_noCache
 
       long long           throttling         = sub.hasField(CSUB_THROTTLING)?       getIntOrLongFieldAsLongF(sub, CSUB_THROTTLING)       : -1;
       long long           lastNotification   = sub.hasField(CSUB_LASTNOTIFICATION)? getIntOrLongFieldAsLongF(sub, CSUB_LASTNOTIFICATION) : -1;
-      std::string         renderFormatString = sub.hasField(CSUB_FORMAT)? getStringFieldF(sub, CSUB_FORMAT) : "";
+      std::string         renderFormatString = sub.hasField(CSUB_FORMAT)? getFieldF(sub, CSUB_FORMAT).String() : "legacy";  // NGSIv1 JSON is 'default' (for old db-content)
```

Comment:
I discover during the fixing of the *noCache test that the logic to retrieve render string may have if the doc doens't have a `format` field. Now, it works as in the csub cache (in fact, this line has been copyed from it).

Not sure if this should be mentioned in CNR. Probably not, as no-`format` would mean "XML". Probably it is only a problem for the existing unittest but not something quite important.

Opinions are welcome :)